docs/zk2kraft-summary.html

```diff
@@ -0,0 +1,27 @@
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one or more
+ contributor license agreements.  See the NOTICE file distributed with
+ this work for additional information regarding copyright ownership.
+ The ASF licenses this file to You under the Apache License, Version 2.0
+ (the "License"); you may not use this file except in compliance with
+ the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<script><!--#include virtual="js/templateData.js" --></script>
+
+<script id="zk2kraft-summary-template" type="text/x-handlebars-template">
+<p>
+    There are many differences between ZooKeeper mode and KRaft mode.
+    The <a href="/{{version}}/zk2kraft.html">Differences Between KRaft mode and ZooKeeper mode</a> page lists all of these differences,
```

Comment:
according to the rule: https://github.com/apache/kafka-site/blob/asf-site/.htaccess#L8

we have to move `zk2kraft.html` to `documentation/zk2kraft.html` to avoid rewriting.